source/documentation/administration_guide/topics/Explanation_of_Settings_in_the_New_Scheduling_Policy_and_Edit_Scheduling_Policy_Window.adoc

```diff
@@ -11,68 +11,84 @@ The following table details the options available in the *New Scheduling Policy*
 |*Description* |A description of the scheduling policy. This field is recommended but not mandatory.
 |*Filter Modules* a|A set of filters for controlling the hosts on which a virtual machine in a cluster can run. Enabling a filter will filter out hosts that do not meet the conditions specified by that filter, as outlined below:
 
+* `ClusterInMaintenance`: Virtual machines being started on the host that are not configured as high availability will filter out the host.
+
 * `CpuPinning`: Hosts which do not satisfy the CPU pinning definition.
 
-* `Migration`: Prevent migration to the same host.
+* `Migration`: Prevents migration to the same host.
+
+* `CPUOverloaded`: Hosts that are CPU overloaded.
 
 * `PinToHost`: Hosts other than the host to which the virtual machine is pinned.
 
 * `CPU-Level`: Hosts that do not meet the CPU topology of the virtual machine.
 
-* `CPU`: Hosts with fewer CPUs than the number assigned to the virtual machine.
+* `VmAffinityGroups`: Hosts that do not meet the conditions specified for a virtual machine that is a member of an affinity group. For example, that virtual machines in an affinity group must run on the same host or on separate hosts.
```

Comment:
@stoobie I "think" what is being conveyed is two things but I agree it's confusing. 

If the affinity group rules state that all VMs within that group must run on the same host and the host does not meet that criteria, it will be filtered out.

or

If the affinity group rule states that all VMs within that group must run on separate hosts and the host does not meet that criteria it will be filtered out. 

Based on feedback from Germano, I've simplified it to "Hosts that do not meet the hard affinity rules defined for the virtual machine."